jstests/noPassthrough/virtual/io_stats_report.js

```diff
@@ -0,0 +1,215 @@
+/**
```

Comment:
I actually renamed `error_report_csv_file.js` into this and added two more test cases but somehow git considers it's one deletion and one addition.